src/crimson/osd/ops_executer.h

```diff
@@ -174,7 +176,9 @@ class OpsExecuter {
       msg(std::move(msg)) {
   }
 
-  osd_op_errorator::future<> execute_op(class OSDOp& osd_op);
+  ::crimson::interruptible::interruptible_errorated_future<
```

Comment:
Again, this just seems really verbose, can we add a using declaration like osd_op_errorator?